src/EnergyPlus/OutAirNodeManager.cc

```diff
@@ -334,24 +336,29 @@ namespace OutAirNodeManager {
 				if ( NumNums > 0 ) Node( NodeNums( 1 ) ).Height = Numbers( 1 );
 
 				if ( NumAlphas > 1 ) {
+					AnyLocalEnvironmentsInModel = true;
+					Node( NodeNums( 1 ) ).IsLocalNode = true;
+				}
+
+				if ( NumAlphas > 1 && !lAlphaBlanks( 2 ) ) {
 					Node( NodeNums( 1 ) ).OutAirDryBulbSchedNum = GetScheduleIndex( Alphas( 2 ) );
```

Comment:
There's no warning here for when a schedule name is not found (i.e., OutAirDryBulbSchedNum = 0).

Reply:
I read this wrong, the warnings were staring me in the face.

Reply:
OK, you just added these warnings. Good.